src/main/java/org/openmrs/isanteplus/performancedata/model/Encounter.java

```diff
@@ -49,23 +46,22 @@
 
     private long visitId;
     
-    private UUID uuid;
+    private String uuid;
 
     @Override
     public String getPreparedSql() {
         return PREPARED_SQL;
     }
 
     @Builder
-    public Encounter(long id, long encounterType, long patientId, long encounterId,
+    public Encounter(long id, long encounterType, long patientId,
                      long locationId, long formId, LocalDateTime encounterDatetime,
                      long creator, LocalDateTime dateCreated, long voided, long voidedBy,
                      LocalDateTime dateVoided, String voidReason, long changedBy,
-                     LocalDateTime dateChanged, long visitId, UUID uuid) {
+                     LocalDateTime dateChanged, long visitId, String uuid) {
```

Comment:
I woudn't change UUID type to string if there is no need. Keeping UUID type helps in avoiding invalid type/value passing

Reply:
No, the tool will stop an execution. I'll add a more precise message for the user.

Reply:
ok